src/Engine/Component/Component.cpp

```diff
@@ -1,5 +1,6 @@
 #include <Engine/Component/Component.hpp>
 
+#include <Core/Math/LinearAlgebra.hpp>

```

Comment:
As already stated, this will highly increase compilation time for `Engine` and client Plugins using `Component`

Reply:
oh yes, my bad